### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by Ibnballo1**
- Page Title: Title changed from "Portfolio setup" to "My Portfolio".
- Image Alt Text: In line 33, image alt text changed from "menu-icon" to "menu".
- Also in lines 273, 277, and 281, the image alt texts changed from "Javascript Icon" to "Javascript", "Html5 icon" to "Html5", and "Css icon" to "CSS"
- Text Headings: No issue found
- Color contrast: changes were made from "#7f8cff" to "#2638dc"
- Resize: No issue found
- Interaction: no issue found
- Moving content: no issue found
- Multimedia: no issue found
- The basic structure of the page: no issue found.

### CODE REVIEW

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @Ibnballo1 and Team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good accessibility report. ✔️

## Required Changes ♻️
- [x] Please add `<label>` for all form input controls. It helps for accessibility, e.g. screen-readers.
As in your design, there's no `label`, You can use hidden attributes for your labels so it will not display but will be available for accessibility.
- [ ] There should be only one h1 tag in a webpage for good SEO, You are using many h1 headings in your project, Please change them with other heading tags example h2, h3, h4, etc and make sure your webpage has only one h1 heading

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ibnballo1** [comment]
Thanks @thecodechaser
I have changed the h1 to h2 leaving just two h1, one for the mobile version and the other for the desktop version.

File: index.html [hunk 1]
```diff
@@ -144,8 +144,8 @@ <h1>Multi-Post Stories</h1>
         <div class="work-card-3">
           <div class="snapshot-portfolio-3"></div>
           <div class="project-title-3">
-            <h1 class="mv-h1">Tonic</h1>
-            <h1 class="dv-h31">Facebook 360</h1>
```
Comment (micheaol) [inline index.html:148]: As requested by the previous code reviewer, there should be only one h1 tag in a webpage for good SEO, You are using many h1 headings in your project, Please change them with other heading tags example h2, h3, h4, etc and make sure your webpage has only one h1 heading

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Ibnballo1 , 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**